### DISCUSSION TITLE
Show notifications for downloads

### DISCUSSION THREAD

**Discussion by SinTan1729** [Radiokot/photoprism-android-client#84] [Ideas]
While downloading, there's currently no notification for download progress. It's fine for small downloads. But if I download multiple pictures or a video, it gets a little annoying since I have to stay on the video screen. Also, the fails to play some videos that play just fine in a browser.

**Comment by Radiokot** [comment]
Hi. You can continue browsing your gallery, the download is done in background. I agree that having a progress notification would be nice.
